src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -0,0 +1,269 @@
+    df_melt_high_outlier = read_outlier(filename_high_melt, get_col_name("melt", "high"))
+    df_wham_high_outlier = read_outlier(filename_high_wham, get_col_name("wham", "high"))
+    df_total_high_outliers = read_all_outlier(df_manta_high_outlier, df_melt_high_outlier, df_wham_high_outlier, "high")
+    df_manta_low_outlier = read_outlier(filename_low_manta, get_col_name("manta", "low"))
+    df_melt_low_outlier = read_outlier(filename_low_melt, get_col_name("melt", "low"))
+    df_wham_low_outlier = read_outlier(filename_low_wham, get_col_name("wham", "low"))
+    df_total_low_outliers = read_all_outlier(df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, "low")
+    df_melt_insert_size = read_melt_insert_size(filename_melt_insert_size)
+
+    # all data frames
+    dfs = [df_ploidy, df_bincov_median, df_wgd_scores, df_non_diploid, df_manta_high_outlier,
+           df_melt_high_outlier, df_wham_high_outlier, df_total_high_outliers,
+           df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, df_total_low_outliers,
+           df_melt_insert_size]
+    output_df = reduce(lambda left, right: pd.merge(left, right, on=ID_COL, how="outer"), dfs)
+    output_df = output_df[output_df[ID_COL] != EMPTY_OUTLIERS]
+
+    # save the file
+    output_df.to_csv(f"{output_prefix}.evidence_qc_table.tsv", sep="\t", header=True, index=False, na_rep=np.nan)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-y", "--estimated-copy-number-filename",
+        help="Sets the filename containing estimated copy numbers per contig.")
+
+    parser.add_argument(
+        "-d", "--median-cov-filename",
+        help="Sets the filename containing median coverage.")
+
+    parser.add_argument(
+        "-g", "--wgd-scores-filename",
+        help="Sets the filename containing WGD scores.")
+
+    parser.add_argument(
+        "-b", "--binwise-cnv-qvalues-filename",
+        help="Sets the filename containing bin-wise CNV q-values.")
+
+    parser.add_argument(
+        "-z", "--manta-qc-outlier-high-filename",
+        help="Sets the filename containing Manta QC outlier high.")
+
+    parser.add_argument(
+        "-e", "--melt-qc-outlier-high-filename",
+        help="Sets the filename containing Melt QC outlier high.")
+
+    parser.add_argument(
+        "-w", "--wham-qc-outlier-high-filename",
+        help="Sets the filename containing Wham QC outlier high.")
+
+    parser.add_argument(
+        "-a", "--manta-qc-outlier-low-filename",
+        help="Sets the filename containing Manta QC outlier low.")
+
+    parser.add_argument(
+        "-s", "--melt-qc-outlier-low-filename",
+        help="Sets the filename containing Melt QC outlier low.")
+
+    parser.add_argument(
+        "-r", "--wham-qc-outlier-low-filename",
+        help="Sets the filename containing Wham QC outlier low.")
+
+    parser.add_argument(
+        "-m", "--melt-insert-size-filename",
```

Comment:
Not necessary to merge.
Why: I think we should update every reference to this file in this script to call it "mean" insert size instead of "Melt". We currently calculate the insert size during MELT, but as we work towards deprecating MELT, that will likely change, so we should be more flexible/accurate with the naming here.
Also: "melt" > "mean" elsewhere in script
```suggestion
        "-m", "--mean-insert-size-filename",
```